examples/osgi-test-example-bnd/cnf/build.bnd

```diff
@@ -0,0 +1,2 @@
+## Place your settings in ext/.
+cnf: ${workspace}/cnf
```

Comment:
Huh? There is already `${build}` macro which does this.

Reply:
removed